templates/src/main/java/org/teavm/flavour/templates/emitting/TemplatingProxyGenerator.java

```diff
@@ -91,7 +91,7 @@ private TemplateInfo parseForModel(ReflectClass<Object> cls, SourceLocation loca
             for (Diagnostic diagnostic : parser.getDiagnostics()) {
                 SourceLocation diagnosticLocation = location != null ? new SourceLocation(location.getMethod(), path,
                         mapper.getLine(diagnostic.getStart()) + 1) : null;
-                diagnostics.error(diagnosticLocation, diagnostic.getMessage());
+                diagnostics.error(diagnosticLocation, path + ": " + diagnostic.getMessage());
```

Comment:
Why this change at all? Diagnostics already takes location (including file name) with location.

Reply:
OK, let me see if I can add the file name to the diagnostic.  No filename is printed when using the 'master' branch code, so maybe it is null right now.

Reply:
For "Property not found", location is null (from TemplatingProxyGenerator:42, which calls Metaprogramming.getLocation(), which apparently returns null).  If location is null, then SourceLocation is null on line 94 and the filename won't be printed

If I try to construct a SourceLocation I don't have a ReflectMethod, do you have any recommendation?

Reply:
In my updated PR it only adds the path if the location is null.  Let me know if there is a better way.